### PR TITLE
Refactor groomer profile form

### PR DESCRIPTION
**Pull request by JessBonanno**
## Description ##

Groomer profile form has been refactored to match the pet-profile form and customer profile form
- broke down the large form into 3 smaller ones for each section including groomer profile, hours of operation, and services offered.
- The hours of operation form was made more user friendly allowing for user to have closed as default when no hours selected .rather than defaulting to 12:00 for open and closed times.
- The closed option will be disabled until a user selects an open time to avoid breaking.
- Loading states have been added when a user submits.
- Created an edit button for individual groomer services but commented out until we discuss if it needs to be hooked up or not.
- The button placement to open the forms is temporary until the groomer dashboard is created and will be moved eventually.

## Type ##

- [ ] This is a bugfix.
- [ ] This is a new feature.
- [ ] This is part of a feature.
- [x] This is a refactor.

## Organization ##

- [x] Have you linked this pull request to a Trello card?
- [x] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 

## Review ##

- [ ] Is this a work in progress?
- [x] Is this ready to be reviewed?
- [x] Have you added two reviewers to this pull request?

### CODE REVIEW

**Comment by JessBonanno** [comment]
[Refactor customer and groomer update profile forms to match the pet profile form once done](https://trello.com/c/rJIZfB7b/49-refactor-customer-and-groomer-update-profile-forms-to-match-the-pet-profile-form-once-done)

**Comment by JessBonanno** [comment]
[Add API call/Function to call services table to populate dropdown in Groomer Profile form where services are added by groomer.](https://trello.com/c/fj6ll0T5/50-add-api-call-function-to-call-services-table-to-populate-dropdown-in-groomer-profile-form-where-services-are-added-by-groomer)

**Review by timotheemcdonald — DISMISSED** [review]
The new refactor looks nice not getting any errors but I'm getting a warning: 
"./src/components/pages/GroomerProfile/ServicesArea.js
  Line 7:26:  'EditOutlined' is defined but never used  no-unused-vars"
- this is an easy fix

**Review by timotheemcdonald — APPROVED** [review]
I'm seeing a warning about strict mode in console but I don't think it's due to these changes so I'm going to approve.

File: src/components/pages/ProfileFormGR/form.scss [hunk 1]
```diff
@@ -1,12 +1,12 @@
 .hours-container{
-    width: 300px;
-    min-height: 300px;
-    border: 1px solid black;
-    margin-bottom: 10px;
-    margin-left: 80px;
+    //width: 300px;
```
Comment (JessicaGCooper) [inline src/components/pages/ProfileFormGR/form.scss:2]: Will we need this commented out code in the future?

**Review by jessicagertig — APPROVED** [review]
I pulled this branch down and all is working. I tested editing groomer info, adding a service, and setting hours for a groomer and all are interacting with the backend as intended.  The UI is significantly improved by separating the 3 sections, changing the default hours to closed, and by formatting of the added service and icon for deleting a service as the button.  Great job!

**Pull request merged**